assignment4/ques12.js

```diff
@@ -0,0 +1,28 @@
+//Write a program to display intersection of two array
+
+let inputArray1 = [1, 2, 3, 4, 5];
+let inputArray2 = [2, 3, 40, 5];
+
+inputArray1.sort((a, b) => a - b);
+inputArray2.sort((a, b) => a - b);
+
+
+function findIntersection(arr1, arr2) {
```

Comment:
findIntersection is not returning any value, so it might return undefined. please check once